api/controller/ReportController.py

```diff
@@ -1,77 +1,169 @@
 from flask import Response, request, render_template
 from api.models.report import Report
 from mongoengine.errors import FieldDoesNotExist, NotUniqueError, DoesNotExist, ValidationError, InvalidQueryError
+from api.helpers.errors import APINotUniqueError, APISchemaError, APIDoesNotExistError
```

Comment:
This file will may have merge conflicts in the next PR because we are currently working on report controller and report model updates